configure.sh

```diff
@@ -56,151 +51,10 @@ fi
 if is_linux; then
   write_to_bazelrc "build:manylinux2010 --crosstool_top=//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.1:toolchain"
```

Comment:
I think we could further simplify `./configure.sh` or completely get rid of it, but let's keep it like this for now so we don't need to change the docs.